runtime/src/lib.rs

```diff
@@ -119,6 +119,11 @@ pub const CENTS: Balance = 1_000 * MILLICENTS;
 pub const DOLLARS: Balance = 100 * CENTS;
 pub const CANS: Balance = CENTS;
 
+// Prints debug output of the `contracts` pallet to stdout if the node is
+// started with `-lruntime::contracts=debug`.
+// Remove this before using on a production chain!
```

Comment:
I think this is debatable. Once the feature is stable you could in theory have this set to `true` since it is used only in the RPC context.

The main cost would be paying for the deployment of a much larger contract size which includes all the debug statements, though there is now nothing to stop you doing that anyway and just not getting your debug messages (as you have just discovered). /cc @athei

Reply:
I don't agree with this comment. The whole point of this API is to have it enabled in production (but only on RPC calls).